ui/js/vocabulary.js

```diff
@@ -1,6 +1,6 @@
 var v_ThreatExchange = {
     URL             : "https://graph.facebook.com/",
-    VERSION         : "v2.0/",
+    VERSION         : "", //"v2.4/",
```

Comment:
This should be version 2.3, the current version. The next version of the API, 2.4, will change some of the functionality for handling descriptors rather than indicators.